src/sass/app_2/_common/base/themes.scss

```diff
@@ -29,6 +29,8 @@ $themes: (
         text_color                       : $COLOR_BLACK,
         text_primary_color               : $COLOR_LIGHT_BLACK_1,
         text_secondary_color             : $COLOR_LIGHT_BLACK_2,
+        tick_step_default                : $COLOR_LIGHT_BLACK_4,
```

Comment:
isn't `tick_step_default` transition is the same like `hover_color`

Reply:
@cakasuma , for me it's a bit asinine to use the same themify rules for a different component/element.  If`hover_color` is updated in the future, it will affect the styling of the `tick` element since I use the same themed `variable`. I'd like to keep the colors separate each component mainly for future-proofing.